### PR TITLE
New Add-on: Pentester Pack

### PR DESCRIPTION
**Pull request by psiinon**
Please comment on the add-ons included, esp if you think any should be added or removed :)

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Comment by DiogoMRSilva** [comment]
Hi, I also would add ascanrule/pscanrules alpha + beta, wappalyzer and json view.

**Comment by DiogoMRSilva** [comment]
> I was wondering about the alpha/beta scan rules, but maybe they could be a separate pack? (We are planning more packs;) Wappalyzer + jsonview are good suggestions too.

Not sure, I think it is something important for a Pentester to have all the rules even if they may have more FPs. But maybe after knowing the other packs it makes more sense. 

The online menu is the web page that allows to make api calls right? If so I would also add it to some other pack related with automation.

**Comment by psiinon** [comment]
Renamed to packpentester and added wappalyzer and jsonview :)

File: addOns/packpentester/src/main/javahelp/help/map.jhm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE map
+  PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp Map Version 1.0//EN"
+         "http://java.sun.com/products/javahelp/map_1_0.dtd">
+
+<map version="1.0">
+    <mapID target="packpentester" url="contents/packpentester.html" />
+    <mapID target="box-pen-icon" url="contents/images/box--pencil.png" />
```
Comment (kingthorin) [inline addOns/packpentester/src/main/javahelp/help/map.jhm:8]: I don't know if it's worth caring but box-pencil/box-pen consistency? (In toc.xml too)

**Comment by thc202** [comment]
Thank you!

**Pull request merged**